FourthTask/figures/types/types.go

```diff
@@ -0,0 +1,70 @@
+//	import $GOPATH/src/github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/figures/types
+//	Package contains structures descriptions and constructors, interface definition
+
+package types
+
+import (
+	"errors"
+	"math"
+)
+
+const (
+	circlePanic   string = " circle can't has negative radius"
+	squarePanic   string = " square can't has negative side"
+	overflowError string = " result will be bigger, that we can coutn. sorry..."
+)
+
+type Figure interface {
+	area() (float64, error)
+	perimetr() (float64, error)
+}
+
+type Circle struct {
+	radius int
+}
+
+func CircleConstruct(newRadius int) (Circle, error) {
```

Comment:
```suggestion
func NewCircle(newRadius int) (Figurine, error) {
```
so, a minor one.  func name could be like `NewStructName` and it must return an interface object `Figurine`